### ISSUE TITLE
is the min limit 10?

### ISSUE THREAD

**Issue by clothiersphere** [thibauts/node-google-search-scraper#8]
I'm trying to limit to 1 or 3 results, 
but no matter what limit under 10 - it still gives back 10.

**Comment by srblum** [comment]
Yes, the minimum is 10, since it's scraping from actual google search results pages, which contain 10 results each.
